frontend/src/components/AdminApplicationOverview/RemoveAllFromSeat.tsx

```diff
@@ -0,0 +1,90 @@
+import KnappBase from 'nav-frontend-knapper';
+import React from 'react';
+import { Redirect } from 'react-router';
+import { IApplication } from '../Application';
+import Modal from '../Modal';
+import './adminApplicationOverview.css';
+import {
+  _PUT_ON_WAITING_LIST,
+  _REMOVE_ALL_APPROVED,
+  _REMOVE_FROM_SEAT,
+  _WAITING_LIST_WARNING,
+} from './strings';
+
+interface IProps {
+  applications: IApplication[];
+  removeFromSeat: (numbers: number[]) => void;
+}
+
+interface IState {
+  modalOpen: boolean;
+}
+
+interface IDispatchProps {
+  getAllApplications: () => void;
+}
+
+type Props = IProps & IDispatchProps;
+
+export class RemoveAllFromSeat extends React.Component<Props, IState> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      modalOpen: false,
+    };
+  }
+
+  public removeAll = () => {
+    const removees = this.props.applications.filter(app => app.seat ? true : false);
+    const thoseWhomWillBeYeetedFromSeat = removees.map(app => app.id);
+    this.props.removeFromSeat(thoseWhomWillBeYeetedFromSeat);
+    this.props.getAllApplications();
+    this.toggleModal();
+    window.location.reload();
```

Comment:
This should be changed.

getAllApplications() wouldn't update the elements, even though it should..